nncf/version.py

```diff
@@ -11,6 +11,6 @@
 
 __version__ = "2.7.0"
 
-BKC_TORCH_VERSION = "2.1.0"
+BKC_TORCH_VERSION = "2.1.2"
```

Comment:
```suggestion
BKC_TORCH_VERSION = "2.1.*"
```
What do you think?

Reply:
`from packaging import version` does not support `*` in version.
`LegacyVersion` and `LegacySpecifier` that support it was deprecated and removed.

Think that using version without wild card it's more correct way.

Reply:
I'm trying to avoid misunderstanding here. You should keep in mind that patch version is not checked. Could you suggest explicitly solution? For example, Using `BKC_TORCH_VERSION = "2.1"` if nncf decelerates support torch~=2.1.0?

My general concern is following. If you need to specify the torch version with exactly patch you will have to fix code again.